README.md

```diff
@@ -22,7 +22,7 @@ protractor-flake
 
 # Full options
 # protractor-flake <protractor-flake-options> -- <options to be passed to protractor>
-protractor-flake --protractor-path=/path/to/protractor --parser standard --node-bin node --max-attempts=3 -- path/to/protractor.conf.js
+protractor-flake --protractor-path=/path/to/protractor --parser standard --node-bin node --max-attempts=3 --color=magenta -- path/to/protractor.conf.js
```

Comment:
Hmm; I still don't see this being super valuable since it is cosmetic. How are you thinking about solving this? If it's something that's easy to do then that's fine; if it's going to add a bunch of additional complexity I don't think it's worth it